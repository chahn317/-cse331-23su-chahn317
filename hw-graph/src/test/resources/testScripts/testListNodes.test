# Tests getNodes()

# Create a graph
CreateGraph graph1

# List nodes in graph1 when empty
ListNodes graph1

# List nodes in graph1 when not empty
AddNode graph1 n1
ListNodes graph1
AddNode graph1 n2
ListNodes graph1
AddNode graph1 n3
ListNodes graph1

# List nodes in graph1 when there are edges
AddEdge graph1 n1 n1 e11
AddEdge graph1 n1 n2 e12
AddEdge graph1 n1 n3 e13
ListNodes graph1
