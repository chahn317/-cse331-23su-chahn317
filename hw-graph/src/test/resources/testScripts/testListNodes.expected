# Tests getNodes()

# Create a graph
created graph graph1

# List nodes in graph1 when empty
graph1 contains:

# List nodes in graph1 when not empty
added node n1 to graph1
graph1 contains: n1
added node n2 to graph1
graph1 contains: n1 n2
added node n3 to graph1
graph1 contains: n1 n2 n3

# List nodes in graph1 when there are edges
added edge e11 from n1 to n1 in graph1
added edge e12 from n1 to n2 in graph1
added edge e13 from n1 to n3 in graph1
graph1 contains: n1 n2 n3